cli/kontena-cli.gemspec

```diff
@@ -22,15 +21,16 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.7"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_runtime_dependency "excon", "~> 0.49.0"
+  spec.add_development_dependency "rake-compiler", "~> 1.0"
+  spec.add_runtime_dependency "excon", ">= 0.49.0", "< 0.50"
   spec.add_runtime_dependency "tty-prompt", "~> 0.10"
-  spec.add_runtime_dependency "clamp", "~> 1.1.0"
-  spec.add_runtime_dependency "ruby_dig", "~> 0.0.2"
-  spec.add_runtime_dependency "launchy", "~> 2.4.3"
-  spec.add_runtime_dependency "hash_validator", "~> 0.7.0"
-  spec.add_runtime_dependency "retriable", "~> 2.1.0"
+  spec.add_runtime_dependency "clamp", ">= 1.1", "< 1.2"
+  spec.add_runtime_dependency "ruby_dig", "< 0.1"
+  spec.add_runtime_dependency "launchy", ">= 2.4.3", "< 2.5"
+  spec.add_runtime_dependency "hash_validator", ">= 0.7", "< 0.8"
+  spec.add_runtime_dependency "retriable", ">= 2.1", "< 2.2"
```

Comment:
That feels like a bad warning, because the suggested variant is totally not equivalent. I guess this is the best way to workaround it, unless there's something to supress it... what about `gem build --force`, is that a bad idea?